app/models/user.rb

```diff
@@ -23,7 +23,7 @@ class User < ApplicationRecord
   include User::CreationOrigin
   include User::Geocodable
 
-  attr_accessor :skip_uniqueness_validations, :import_associations_from_rdv_solidarites_on_create
+  attr_accessor :skip_uniqueness_validations, :import_associations_from_rdv_solidarites_on_create, :validate_title
```

Comment:
nitpick: `validate_title` on dirait plus un nom de méthode de validation IMO, on pourrait faire
```suggestion
  attr_accessor :skip_uniqueness_validations, :import_associations_from_rdv_solidarites_on_create, :require_title_presence
```